addon/components/editor-plugins/citations/modal.hbs

```diff
@@ -58,48 +58,48 @@
               </div>
               <div class="say-form__row">
                 <label class="say-label say-label--block" for="search-codex-datum-document-van">Filter op datum document</label>
-                <div class="form__row">
+                <div class="form__row" style="display: flex">
```

Comment:
if you're using inline styles, please ignore the error for eslint using
```
{{!-- template-lint-configure no-inline-styles "warn" --}}>
```

Reply:
I think we want to include that inline style into a class instead of ignoring the error, that's why I asked Dietr